src/apps/material/helper.ts

```diff
@@ -402,3 +410,75 @@ export const isParallelReservation = (manifestations: Manifestation[]) =>
   manifestations.length > 1 &&
   hasCorrectAccessType(AccessTypeCode.Physical, manifestations) &&
   !isArticle(manifestations);
+
+// Because we  need to exclude the branches that are blacklisted, we need to use a custom hook to prevent duplicate code
+const getBlacklistedArgs = (
```

Comment:
I think it should be called `getBlacklistedQueryArgs` to make it clear what it is being used for. I needed some extra seconds to decipher the meaning of it.